<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchiveItem</class>
 <widget class="QWidget" name="ArchiveItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>15</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>My Farm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_edit_time">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>55</y>
     <width>241</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apr 25, 2022</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_difficulty">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>55</y>
     <width>181</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIfficulty</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
